ibm/resource_ibm_network_gateway_test.go

```diff
@@ -0,0 +1,138 @@
+package ibm
+
+import (
+	"fmt"
+	//"log"
+	"strconv"
+	//"strings"
+	//"time"
+	"errors"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	//"github.com/hashicorp/terraform/helper/schema"
+	"github.com/softlayer/softlayer-go/datatypes"
+	//"github.com/softlayer/softlayer-go/filter"
+	//"github.com/softlayer/softlayer-go/helpers/location"
+	//"github.com/softlayer/softlayer-go/helpers/product"
```

Comment:
Probably a nitpick, let us remove all these unwanted packages altogether. I am not if you are already using some good IDE which will take care of it automatically when you save the file.